AdaptiveUiCodelab/app/src/main/java/com/example/reply/ui/MainActivity.kt

```diff
@@ -34,6 +36,11 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        enableEdgeToEdge()
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
Haven't seen this done before, should we add it to the compose docs for edge to edge?

Reply:
Yes, created a doc CL to talk about removing background protection for 3 button nav